.github/workflows/ci-app.yml

```diff
@@ -0,0 +1,77 @@
+name: Continuous Integration Delta App
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'delta/sourcing/**'
+      - 'delta/rdf/**'
+      - 'delta/sdk/**'
+      - 'delta/sdk-testkit/**'
+      - 'delta/service/**'
+      - 'delta/app/**'
+      - 'build.sbt'
+      - 'project/**'
+  pull_request:
+    paths:
+      - 'delta/sourcing/**'
+      - 'delta/rdf/**'
+      - 'delta/sdk/**'
+      - 'delta/sdk-testkit/**'
+      - 'delta/service/**'
+      - 'delta/app/**'
+      - 'build.sbt'
+      - 'project/**'
+jobs:
+  review:
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+    steps:
+      - uses: actions/checkout@v2.1.0
+      - name: Cache Coursier
+        uses: actions/cache@v1.1.2
+        with:
+          path: ~/.cache/coursier
+          key: ${{ runner.os }}-coursier-scala-${{ hashFiles('**/*.sbt') }}
+          restore-keys: ${{ runner.os }}-coursier-scala-
+      - uses: olafurpg/setup-scala@v5
+        with:
+          java-version: adopt@1.11
+      - name: StaticAnalysis
+        run: sbt -Dsbt.color=always -Dsbt.supershell=false "project app" clean scalafmtCheck test:scalafmtCheck scalafmtSbtCheck scapegoat
+      - name: Tests
+        run: sbt -Dsbt.color=always -Dsbt.supershell=false "project app" clean coverage test coverageReport coverageAggregate
+  publish:
+    if: github.event_name == 'push'
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+    env:
+      CI: true
+      BINTRAY_USER: ${{ secrets.BINTRAY_USER }}
+      BINTRAY_PASS: ${{ secrets.BINTRAY_PASS }}
+    steps:
+      - uses: actions/checkout@v2.1.0
+      - name: Cache Coursier
+        uses: actions/cache@v1.1.2
+        with:
+          path: ~/.cache/coursier
+          key: ${{ runner.os }}-coursier-scala-${{ hashFiles('**/*.sbt') }}
+          restore-keys: ${{ runner.os }}-coursier-scala-
+      - uses: olafurpg/setup-scala@v5
+        with:
+          java-version: adopt@1.11
+    - name: PublishToBintray
+      run: sbt -Dsbt.color=always -Dsbt.supershell=false "project app" releaseEarly
+    - name: PublishToDockerHub
+      run: |
+        sbt -Dsbt.color=always -Dsbt.supershell=false "project app" docker:publishLocal
+        echo ${{ secrets.DOCKER_PASS }} | docker login --username ${{ secrets.DOCKER_USER }} --password-stdin
+        docker push bluebrain/nexus-delta-app:latest
```

Comment:
We should have `bluebrain/nexus-delta` here and to modify the build.sbt to set the correct docker image name.

Reply:
For now I will comment this part. Let's deal with it afterwards